pkg/ebpf/tracee.go

```diff
@@ -59,6 +60,45 @@ type fileExecInfo struct {
 	Hash      string
 }
 
+// fileUniqueId is an unique identifier for a file, based on its device and inode.
+type fileUniqueId struct {
+	dev uint64
+	ino uint64
+}
+
+// Stat attempts to retrieve the device and inode numbers from the given path.
+// This function should be called immediately after obtaining the path.
+// This ensures that even if the path is lost or changes, that file can still
+// be identified and accessed using its device and inode numbers.
+func (f *fileUniqueId) Stat(path string) error {
+	onceHashCapsAdd.Do(
+		func() {
+			err := capabilities.GetInstance().BaseRingAdd(cap.SYS_PTRACE)
```

Comment:
I know that you didn't implement it, but I am just checking if someone knows - don't we also need `CAP_DAC_OVERRIDE` for this?